github/resource_github_repository_deploy_key_test.go

```diff
@@ -11,21 +13,43 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+func TestSuppressDeployKeyDiff(t *testing.T) {
+	oldV := "ssh-rsa AAAABB...cd+== terraform-acctest@hashicorp.com\n"
+	newV := "ssh-rsa AAAABB...cd+=="
+	if suppressDeployKeyDiff("test", oldV, newV, nil) {
+		t.Fatalf("Expected %q and %q to be suppressed", oldV, newV)
```

Comment:
did you mean NOT suppressed in the error message?

Reply:
This is the only real blocker for me on the PR, either the message is missing `NOT` or the conditional should be negated.

Reply:
You're right, the reason the test was passing is because I accidentally swapped `oldV` and `newV`. 🙈 
Fixed now!